docs/package.json

```diff
@@ -56,8 +56,6 @@
     "codelyzer": "^5.0.1",
     "protractor": "^5.3.2",
     "stylelint": "7.13.0",
-    "tslint": "5.8.0",
```

Comment:
Can it be the same for the angular and react applications?

Reply:
yes (because we used nebular tslint config 😄 )